package.json

```diff
@@ -17,65 +17,34 @@
     "tailwind:build": "tailwind-config-viewer export ./tailwind-static",
     "tailwind:view": "tailwind-config-viewer -o"
   },
-  "dependencies": {},
   "devDependencies": {
+    "@babel/core": "^7.13.8",
+    "@babel/plugin-proposal-numeric-separator": "^7.14.5",
     "@balancer-labs/assets": "github:balancer-labs/assets#master",
     "@balancer-labs/balancer-js": "^1.0.1",
     "@balancer-labs/sor": "^1.1.0",
     "@balancer-labs/sor2": "npm:@balancer-labs/sor@^2.0.0-beta.8",
     "@balancer-labs/typechain": "^1.0.0",
     "@ensdomains/content-hash": "^2.5.3",
+    "@ethersproject/abi": "~5.4.1",
+    "@ethersproject/abstract-provider": "~5.4.1",
+    "@ethersproject/abstract-signer": "~5.4.1",
+    "@ethersproject/address": "~5.4.0",
+    "@ethersproject/bignumber": "~5.4.2",
+    "@ethersproject/constants": "~5.4.0",
+    "@ethersproject/contracts": "~5.4.1",
+    "@ethersproject/providers": "~5.4.4",
+    "@georgeroman/balancer-v2-pools": "^0.0.5",
     "@gnosis.pm/gp-v2-contracts": "^1.0.2",
     "@gnosis.pm/safe-apps-provider": "^0.6.0",
     "@gnosis.pm/safe-apps-sdk": "4.0.0",
-    "@georgeroman/balancer-v2-pools": "^0.0.5",
     "@metamask/detect-provider": "^1.2.0",
     "@popperjs/core": "^2.9.2",
     "@portis/web3": "^3.0.10",
     "@sentry/browser": "^6.3.2",
     "@sentry/tracing": "^6.3.2",
-    "@types/animejs": "^3.1.3",
-    "@vueuse/core": "^4.3.4",
-    "@walletconnect/web3-provider": "^1.5.2",
-    "animejs": "^3.2.1",
-    "awesome-only-resolves-last-promise": "^1.0.3",
-    "axios": "0.21.2",
-    "bignumber.js": "^9.0.1",
-    "bnc-sdk": "^3.5.0",
-    "color.js": "^1.2.0",
-    "date-fns": "^2.21.1",
-    "echarts": "^5.2.0",
-    "ethereumjs-util": "^7.0.7",
-    "feather-icons": "^4.28.0",
-    "json-to-graphql-query": "^2.0.0",
-    "lodash": "^4.17.15",
-    "numeral": "^2.0.4",
-    "paraswap": "^4.18.0",
-    "parse-filepath": "1.0.2",
-    "swiper": "^6.4.5",
-    "vue": "3.2.6",
-    "vue-composable-tester": "^0.1.3",
-    "vue-echarts": "^6.0.0",
-    "vue-i18n": "^9.0.0-rc.1",
-    "vue-query": "^1.6.0",
-    "vue-router": "^4.0.10",
-    "vue-slider-component": "^4.0.0-beta.3",
-    "vue3-jazzicon": "^0.1.2",
-    "vue3-virtual-scroller": "^0.2.1",
-    "vuex": "^4.0.2",
-    "walletlink": "^2.1.5",
-    "web3-utils": "^1.3.1",
-    "@ethersproject/abi": "~5.4.1",
-    "@ethersproject/address": "~5.4.0",
-    "@ethersproject/bignumber": "~5.4.2",
-    "@ethersproject/constants": "~5.4.0",
-    "@ethersproject/contracts": "~5.4.1",
-    "@ethersproject/providers": "~5.4.4",
-    "@ethersproject/abstract-provider": "~5.4.1",
-    "@ethersproject/abstract-signer": "~5.4.1",
-    "@babel/core": "^7.13.8",
-    "@babel/plugin-proposal-numeric-separator": "^7.14.5",
```

Comment:
I am pretty sure no versions are changed here. The huge package json diff is probably caused by npm trying to rearrange everything in alpha order?

Reply:
yep seems so